Assets/Scripts/Bosses/MissileTelegraph.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MissileTelegraph : MonoBehaviour
+{
+    [SerializeField] float duration;
+    float spawnTime;
+
+    void Start()
+    {
+        spawnTime = Time.time;
```

Comment:
This is extremely minor, but this method is subject to floating point precision loss.

Starting at a value of `0` would be a way to avoid this. Probably doesn't matter for this gamejam though.

Reply:
e.g.

```
float timeElapsed = 0f;

void Update() {
  timeElapsed += Time.deltaTime;
  if (timeElapsed > duration) Splode();
}
```

Reply:
Not sure if I understand the suggestion. How would we keep track of the time since spawning if we set the spawnTime to zero?

Reply:
`timeElapsed` would be the time since spawning. It will initialize to `0` and increment by `Time.deltaTime` each frame. And then continue incrementing until it is equal to or greater than `duration`.

Reply:
Ignore my last comment, for some reason your second comment wasn't displaying when I wrote that. This makes sense